presto-main/src/test/java/com/facebook/presto/operator/OperatorAssertion.java

```diff
@@ -62,6 +63,21 @@ private OperatorAssertion()
 
     public static List<Page> toPages(Operator operator, Iterator<Page> input)
     {
+        try {
+            return toPagesUnsafe(operator, input);
+        }
+        finally {
+            try {
+                operator.close();
```

Comment:
I understand this is somewhat the simplest way to make sure that operators are closed. But it feels rather strange that a method called `toPages` would close the operator. Additionally, if you are to do this here, you should also do it for `public static List<Page> toPages(Operator operator)`.